src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -56,187 +60,130 @@ static constexpr std::string_view AntialiasingModeKey{ "antialiasingMode" };
 static constexpr std::string_view TabColorKey{ "tabColor" };
 
 Profile::Profile() :
-    Profile(std::nullopt)
+    Profile(nullptr)
 {
 }
 
-Profile::Profile(const std::optional<GUID>& guid) :
-    _guid(guid),
-    _name{ L"Default" },
-    _schemeName{ L"Campbell" },
-    _hidden{ false },
-
-    _defaultForeground{},
-    _defaultBackground{},
-    _selectionBackground{},
-    _cursorColor{},
-    _tabTitle{},
-    _suppressApplicationTitle{},
-    _historySize{ DEFAULT_HISTORY_SIZE },
-    _snapOnInput{ true },
-    _altGrAliasing{ true },
-    _cursorShape{ CursorStyle::Bar },
-    _cursorHeight{ DEFAULT_CURSOR_HEIGHT },
-
-    _connectionType{},
-    _commandline{ L"cmd.exe" },
-    _startingDirectory{},
-    _fontFace{ DEFAULT_FONT_FACE },
-    _fontSize{ DEFAULT_FONT_SIZE },
-    /* _fontWeight is initialized below because the structure won't accept a uint16_t directly */
-    _acrylicTransparency{ 0.5 },
-    _useAcrylic{ false },
-    _scrollbarState{},
-    _closeOnExitMode{ CloseOnExitMode::Graceful },
-    _padding{ DEFAULT_PADDING },
-    _icon{},
-    _backgroundImage{},
-    _backgroundImageOpacity{},
-    _backgroundImageStretchMode{},
-    _backgroundImageAlignment{},
-    _retroTerminalEffect{},
-    _antialiasingMode{ TextAntialiasingMode::Grayscale }
+Profile::Profile(IReference<guid> guid) :
+    _Guid(guid ? static_cast<decltype(_Guid)>(guid.Value()) : std::nullopt)
 {
+    /* FontWeight is initialized here because the structure won't accept a uint16_t directly */
     winrt::Windows::UI::Text::FontWeight weight;
     weight.Weight = DEFAULT_FONT_WEIGHT;
-    _fontWeight = weight;
+    _FontWeight = weight;
 }
 
 Profile::~Profile()

```

Comment:
If it has no body, why is it explicitly defined? Can't we let the default one apply? (This would violate rule of 5)